### ISSUE TITLE
Tweak histology ranges in multiple SEER recode algorithms

### ISSUE THREAD

**Issue by depryf** [imsweb/algorithms#176]
Site Recode ICD-O-3/WHO 2008 Definition updates for new histology codes: 

33042: change the range of 9811-9818 to 9811-9819
35011: change the range of 9811-9818 to 9811-9819 
35021: add 9877-9879
35021: change 9910-9911 to 9910-9912 
37000: change 9750-9769 to 9749-9769
37000: change 9965-9967 to 9965-9968
37000: change 9991-9992 to 9991-9993 

Any range that ends in 9992 should be changed to 9993.

**Comment by depryf** [comment]
This is done.

I also renamed the constant SeerSiteREcodeUtils.VERSION_2010 to VERSION_2008 to better reflect its value (I didn't change the actual value).

**Comment by depryf** [comment]
We need to tweak the histology ranges for more SEER algorithms. Here is a full summary:

**Site Recode ICD-O-3/WHO 2008 Definition** (https://seer.cancer.gov/siterecode/icdo3_dwhoheme/index.html)

```
33042: change the range of 9811-9818 to 9811-9819
35011: change the range of 9811-9818 to 9811-9819
35021: add 9877-9879
35021: change 9910-9911 to 9910-9912
37000: change 9750-9769 to 9749-9769
37000: change 9965-9967 to 9965-9968
37000: change 9991-9992 to 9991-9993

Any range that ends in 9992 should be changed to 9993.
````

**ICCC Recode Third Edition ICD-O-3/IARC 2017** (https://seer.cancer.gov/iccc/iccc-iarc-2017.html)

```
011: 9811-9818, 9837 -> 9811-9819, 9837 (added 9819)
012: 9840, 9861, 9865-9867, 9869-9874, 9891, 9895-9897, 9898, 9910, 9911, 9920, 9930, 9931 -> 9840, 9861, 9865-9867, 9869-9874, 9877-9879, 9891, 9895-9897, 9898, 9910, 9911, 9912, 9920, 9930, 9931 (added 9877-9879 and 9912)
014: 9945, 9946, 9975, 9980, 9982-9987, 9989, 9991, 9992 -> 9945, 9946, 9975, 9980, 9982-9987, 9989, 9991, 9993 (added 9993)
015: 9800, 9801, 9805-9809, 9860, 9965-9967 -> 9800, 9801, 9805-9809, 9860, 9965-9968 (added 9968)
022: 9811-9818, 9837 -> 9811-9819, 9837 (added 9819)
022: 9700-9702, 9705, 9708, 9709, 9714, 9716, 9716-9719, 9724-9726, 9767, 9768 -> 9700-9702, 9705, 9708, 9709, 9714, 9716-9719, 9724-9726, 9767, 9768 (removed 9716)
024: 9740-9742, 9750, 9751, 9754-9759 -> 9740-9742, 9749, 9750, 9751, 9754-9759 (added 9749)
094: 8587, 8710-8714, 8806, 8840-8842, 8921, 8990, 8992, 9045, 9373 -> 8587, 8710-8714, 8806, 8840-8842, 8921, 8990, 8992, 9045, 9366-9368, 9373 (added 9366-9368)
104: 8010-8041, 8044, 8050-8075, 8082, 8120-8123, 8130-8141, 8143, 8153, 8190-8201, 8210, 8211, 8213, 8221-8241, 8244-8246, 8260-8263, 8290, 8310, 8320, 8323, 8380-8384, 8430, 8440-8442, 8450, 8470, 8480-8490, 8504, 8510, 8550, 8560-8562, 8570-8573, 9000, 9014 -> 8010-8041, 8044, 8050-8075, 8082, 8120-8123, 8130-8141, 8143, 8153, 8190-8201, 8210, 8211, 8213, 8221-8241, 8244-8246, 8260-8263, 8290, 8310, 8320, 8323, 8380-8384, 8430, 8440-8442, 8450, 8470, 8480-8490, 8504, 8510, 8550, 8560-8562, 8570-8573, 9000, 9014, 9111 (added 9111)
112: 8010-8041, 8050-8075, 8082, 8120-8122, 8130-8141, 8158, 8190, 8200, 8201, 8211, 8230, 8231, 8244-8246, 8260-8263, 8290, 8310, 8320, 8323, 8324, 8333, 8430, 8440, 8480, 8481, 8510, 8560-8562, 8570-8573, 8589 -> 8010-8041, 8050-8075, 8082, 8120-8122, 8130-8141, 8158, 8190, 8200, 8201, 8211, 8230, 8231, 8244-8246, 8260-8263, 8290, 8310, 8320, 8323, 8324, 8333, 8430, 8440, 8480, 8481, 8510, 8560-8562, 8570-8573, 8589, 9111 (added 9111)
112: 8330-8332, 8334-8337, 8340-8347, 8350, 8588 -> 8330-8332, 8334-8337, 8340-8349, 8350, 8588 (added 8348 and 8349)
113: 8010-8041, 8050-8075, 8082, 8083, 8120-8122, 8130-8141, 8190, 8200, 8201, 8211, 8230, 8231, 8244-8246, 8260-8263, 8290, 8310, 8320, 8323, 8430, 8440,8480, 8481, 8500-8551, 8560-8562, 8570-8576 -> 8010-8041, 8050-8075, 8082, 8083, 8120-8122, 8130-8141, 8190, 8200, 8201, 8211, 8230, 8231, 8244-8246, 8260-8263, 8290, 8310, 8320, 8323, 8430, 8440,8480, 8481, 8500-8551, 8560-8562, 8570-8576, 9111 (added 9111)
115: 8010-8041, 8050-8075, 8078, 8081, 8082, 8090-8098, 8100-8110, 8140, 8143, 8147, 8190, 8200, 8211, 8240, 8246, 8247, 8260, 8263, 8310, 8320, 8323, 8390-8420, 8430, 8480, 8540, 8542, 8560, 8570-8573 -> 8010-8041, 8050-8075, 8078, 8081, 8082, 8090-8098, 8100-8110, 8140, 8143, 8147, 8190, 8200, 8211, 8240, 8246, 8247, 8260, 8263, 8310, 8320, 8323, 8390-8420, 8430, 8480, 8540, 8542, 8560, 8570-8573, 9111 (added 9111)
116: 8010-8077, 8080-8086, 8098, 8120-8158, 8163, 8190-8265, 8290, 8310, 8314-8315, 8320-8324, 8333, 8360, 8380-8384, 8401, 8402, 8405, 8406, 8410, 8430-8442, 8450, 8452-8454, 8460, 8461, 8470, 8480-8586, 8589, 8941, 8983, 9000, 9010-9014, 9016, 9020, 9030 -> 8010-8077, 8080-8086, 8098, 8120-8158, 8163, 8190-8265, 8290, 8310, 8314-8315, 8320-8324, 8333, 8360, 8380-8384, 8401, 8402, 8405, 8406, 8410, 8430-8442, 8450, 8452-8455, 8460, 8461, 8470, 8480-8586, 8589, 8941, 8983, 9000, 9010-9014, 9016, 9020, 9030, 9111 (added 8455 and 9111)
121: 8930-8935, 8940, 8950, 8951, 8974, 8980-8982 -> 8930-8935, 8940, 8950, 8951, 8974, 8976, 8980-8982 (added 8976)
```

**Lymphoid Neoplasm Recode 2021 Revision** (https://seer.cancer.gov/lymphomarecode/lymphoma-2021.html)

```
Change range 9811-9818 to 9811-9819 in code 07
Add 9715 to code 33
```

**Brain/CNS Recode** (https://seer.cancer.gov/seerstat/variables/seer/brain_cns-recode/)

```
Change 9590-9992 to 9590-9993 which will affect groups 15 and 24
Add 9563/0 to group 17
Add 9509/3 to grouip 14
9509/0 should already be in group 20
```

**Comment by depryf** [comment]
All those changes are done, they will be released with version 4.6.
